ofl/notoserifjp/README

```diff
@@ -0,0 +1,11 @@
+This package is part of the noto project.  Visit
+google.com/get/noto for more information.
+
+Built on 2017-10-24 from the following noto repository:
+-----
+Repo: noto-cjk
+Tag: v2017-06-01-serif-cjk-1-1
+Date: 2017-09-20 09:49:40 GMT
+Commit: 32a5844539f2e348ed36b44e990f9b06d7fb89fe
+
+Update serif CJK to 1.1.
```

Comment:
Again I think perhaps you don't want to include the READMEfiles

Reply:
Thanks for flagging.